### ISSUE TITLE
Only send reminders to existing customers

### ISSUE THREAD

**Issue by boboldehampsink** [mediabeastnz/craft-commerce-abandoned-cart#86]
### What are you trying to do?

According to German law, abandoned cart emails may only be sent to already existing customers. 

### What's your proposed solution?

Could we have a config option that checks if an email address already has a completed order as well?

### Additional context

It would be even better if this was configurable per site, so that you could, for example, enable this for a german site only.

**Comment by engram-design** [comment]
I was going to ask how we define an existing user. Just a Craft user with that same email, or that another (completed) order exists for that email. I'm thinking the latter?

**Comment by boboldehampsink** [comment]
Yeah I think the latter too. I think technically you need some sort of consent to send these emails. I was told being an existing customer was enough, unless there is some other way...

**Comment by engram-design** [comment]
Added for the next release. To get this early, run `composer require verbb/abandoned-cart:"dev-craft-4 as 3.0.3"` or `composer require verbb/abandoned-cart:"dev-craft-5 as 4.0.3"`.

**Comment by boboldehampsink** [comment]
@engram-design any eta for release?

**Comment by engram-design** [comment]
Fixed in [3.0.4](https://github.com/verbb/abandoned-cart/releases/tag/3.0.4) and [4.0.4](https://github.com/verbb/abandoned-cart/releases/tag/4.0.4)
